src/UIView/UIView+Constraints.m

```diff
@@ -105,22 +105,50 @@ - (void)constrainView:(UIView *)view toHeight:(CGFloat)height {
     [self addConstraint:heightConstraint];
 }
 
+- (void)constrainView:(UIView *)view toAspectRatio:(CGFloat)aspectRatio {
```

Comment:
I like the functionality of this, but the other methods are using `multiplier`, and I think we should stick with that naming convention, maybe something like `withModifier:`